_layouts/devcamp.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+paginate: 5
+---
+
+<p>
+    <center><img src="/images/devcamp.svg" alt="devcamp" width="200px"></center>
+    <br>
+</p>
+
+{{content}}
+
+{% for event in site.categories.devcamp %}
+{% if forloop.first %}
+<ul class="devcamp">
+{% endif %}
+    <li class="devcamp">
+        <a href="{{ event.url }}" title="{{ event.title }}">{{ event.title }}</a>
+    </li>
+{% if forloop.last %}
+</ul>
+{% endif %}
+{% endfor %}
+
+<hr>
+<p>
+    <small><em>Le logo ci-dessus est de Kokota sous licence pour the Noun Project<a rel="license" href="http://creativecommons.org/licenses/by/3.0/us/">Creative Commons Attribution 3.0 United States License</a>.</em></small>
```

Comment:
"sous licence CC pour The Noun Project" non ?

Reply:
ah oui j'ai inversé la phrase !